### ISSUE TITLE
Design & Implement domain object to represent classification results

### ISSUE THREAD

**Issue by rahul-imaging** [Project-MONAI/monai-deploy-app-sdk#19]

**Comment by dbericat** [comment]
Agreed on App SDK sub WG to get 3-4 examples of classification results and try to come up with a template that provides for the most common attributes. 

If not, value pair?

**Comment by MMelQin** [comment]
> Agreed on App SDK sub WG to get 3-4 examples of classification results and try to come up with a template that provides for the most common attributes.
> 
> If not, value pair?
> 
> @CPBridge @vikashg @tomaroberts @laurencejackson @MMelQin
@rahul-imaging 
I've dealt with more than that many examples already, text, csv, json, html table, but more are welcome.

Anyone can have their own way of saving results. The key issue is how to use this representation for the downstream processing, e.g. result writing into DICOM objects. You either have simplistic dictionary and transform it in the writer (not good), or have a customer transformer in between (preferred and what I recommended App SDK users do).

In any case, the KEY ISSUE is not as much in the formats of the classification results are in, rather in making CLINICAL SENSE of the magic strings. In more specific terms, finding and matching the CODES, as called for by DICOM objects and required by the writers, e.g. highdicom.
